src/layouts/MainLayout.js

```diff
@@ -41,9 +42,21 @@ const MainLayout = ({ children, pageContext, sidebarOpen = true }) => {
     }
   };
 
+  /*
+   * [VSU] some docs pages are being designed as JS for faster experimenting
+   * and will never have the frontmatter property
+   * Using regex for check to account for paths with and without trailing slash
+   */
+  const docsAsJS = ['/introduction-apm'];
+  docsAsJS.forEach((docUrl) => {
+    if (location.pathname.match(docUrl) && !hideNavs) {
+      hideNavs = true;
+    }
+  });
```

Comment:
how about this?
```suggestion
  const isJSDoc = docsAsJS.some((docUrl) => docUrl.test(location.pathname));
  hideNavs ||= isJSDoc
```

`String#match` would work here, but it returns either an array of matches or `null`, whereas `RegExp#test` returns a boolean, which feels more appropriate here